stdlib/tests/mem/mod.rs

```diff
@@ -128,6 +128,7 @@ fn test_pipe_words_to_memory() {
             push.1  # number of words
 
             exec.mem::pipe_words_to_memory
+            dropw swapw dropw
```

Comment:
Should we replace this with `exec.rpo::squeeze_digest`. Also, maybe we should do a search&replace for `dropw swapw dropw` sequence to fix it in other places as well.

Reply:
done